include/albatross/src/indexing/apply.hpp

```diff
@@ -54,6 +40,70 @@ auto apply(const std::vector<ValueType> &xs, const ApplyFunction &f) {
   return output;
 }
 
+// Map
+
+template <
+    template <typename...> class Map, typename KeyType, typename ValueType,
+    typename ApplyFunction,
+    typename ApplyType = typename details::key_value_apply_result<
+        ApplyFunction, KeyType, ValueType>::type,
+    typename std::enable_if<details::is_valid_key_value_apply_function<
+                                ApplyFunction, KeyType, ValueType>::value &&
+                                std::is_same<void, ApplyType>::value,
+                            int>::type = 0>
+void apply(const Map<KeyType, ValueType> &map, const ApplyFunction &f) {
```

Comment:
Shouldn't these functions be declared `inline`? Probably true for all of them in this file.

Reply:
Yes!  Nice catch.